src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/runners/RcaItRunnerBase.java

```diff
@@ -267,6 +267,12 @@ private void applyMethodLevelAnnotationOverrides(Method method) throws Exception
                 failedValidations.add(validator.getClass());
               }
               break;
+            case SQLITE:
+              successful = validator.check(testApi.constructObjectFromDBOnHost(expect.on(), rca));
+              if (!successful) {
+                failedValidations.add(validator.getClass());
+              }
```

Comment:
Because that piece is common, do we wan to move it to the else block of `if (successful) {` ?